modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/ComponentMessageProcessor.java

```diff
@@ -759,12 +757,9 @@ private void setOperationExecutionParams(ComponentLocation location, CoreEvent e
   }
 
 
-  private void prepareAndExecuteOperation(CoreEvent event, Supplier<ExecutorCallback> callbackSupplier, Context ctx) {
+  private void prepareAndExecuteOperation(CoreEvent event, Supplier<ExecutorCallback> callbackSupplier) {
     OperationExecutionParams oep = from(event).getOperationExecutionParams(getLocation(), event.getContext().getId());
 
-    final Scheduler currentScheduler = (Scheduler) ctx.getOrEmpty(PROCESSOR_SCHEDULER_CONTEXT_KEY)
```

Comment:
isn't this affecting the threading when handling retries?

Reply:
what do you mean? this scheduler is never used